DataFormats/MuonReco/src/Muon.cc

```diff
@@ -213,6 +249,21 @@ int Muon::numberOfMatchedRPCLayers( ArbitrationType type ) const
    return layers;
 }
 
+
+int Muon::numberOfMatchedME0Layers( ArbitrationType type ) const
+{
+   int layers(0);
+
+   unsigned int theME0LayerMask = ME0layerMask(type);
```

Comment:
This doesn't make much sense considering that ME0 segments are reconstructed in the full ME0 (single station).
Layers are needed if the match ID is to be computed in individual layers (as RPCs are matched separately)